plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/ReadRowsHelper.java

```diff
@@ -81,10 +85,12 @@ public ReadRowsResponse next()
             do {
                 try {
                     ReadRowsResponse response = serverResponses.next();
+                    log.debug("ReadRowsResponse from BigQuery: %s", response);
                     nextOffset += response.getRowCount();
                     return response;
                 }
                 catch (Exception e) {
+                    log.warn(e, "Exception happened during iterating server response");
```

Comment:
logging a stacktrace and retrying isn't good, as this may easily flood logs, and with something that's not actionable.

if the attempt is retried, this should probably go on debug
if the attempt is not retried, this doesn't need to be logged at all

Reply:
Updated.